src/masternodes/rpc_proposals.cpp

```diff
@@ -449,7 +449,8 @@ UniValue votegov(const JSONRPCRequest &request) {
     RPCTypeCheck(request.params, {UniValue::VSTR, UniValue::VSTR, UniValue::VSTR, UniValue::VARR}, true);
 
     auto propId = ParseHashV(request.params[0].get_str(), "proposalId");
-    auto mnId   = ParseHashV(request.params[1].get_str(), "masternodeId");
+    std::string id = request.params[1].get_str();
+    uint256 mnId = uint256();
```

Comment:
Note that you only need as uint256 will default initialise itself as the class has a default constructor.

`uint256 mnId;`